client/my-sites/marketplace/pages/marketplace-product-install/style.scss

```diff
@@ -40,6 +25,10 @@ body.is-section-marketplace.theme-default.color-scheme {
 	justify-content: center;
 	height: 100vh;
 	align-items: center;
+	position: absolute;
+	top: 0;
+	right: 0;
+	left: 0;
```

Comment:
Any particular reason why this is needed?

Reply:
These rules were added to ensure the progress bar is vertically centre aligned. Without these rules, the progress bar is not centred:

<img width="1009" alt="Screen Shot 2024-01-29 at 9 01 08 AM" src="https://github.com/Automattic/wp-calypso/assets/104910361/1590fb37-6cb9-4ab9-a49d-014bfb3da424">